docs/markdown/Reference-manual.md

```diff
@@ -786,6 +786,40 @@ The following keyword arguments are supported:
 - `exclude_directories`: a list of directory names that should not be installed.
   Names are interpreted as paths relative to the `subdir_name` location.
 - `install_dir`: the location to place the installed subdirectory.
+- `strip_directory`: install directory contents. `strip_directory=false` by default.
+  If `strip_directory=false` only last component of source path is used.
+  Since 0.45.0
+
+For a given directory `foo`:
+```
+foo/
+  bar/
+    file1
+  file2
+```
+`install_subdir('foo', install_dir : 'share', strip_directory : false)` creates
+```
+share/
+  foo/
+    bar/
+      file1
+    file2
+```
+
+`install_subdir('foo', install_dir : 'share', strip_directory : true)` creates
+```
+share/
+  bar/
+    file1
+  file2
+```
+
+`install_subdir('foo/bar', install_dir : 'share', stripe_directory : false)` creates
```

Comment:
There is an extra 'e' in `stripe_directory`. And since this needs fixing there should also be an example of installing `foo/bar` with `strip_directory` set to true, which results in `share/file1` and nothing else. Thanks.

Reply:
Fixed.